content/actions/using-workflows/workflow-syntax-for-github-actions.md

```diff
@@ -99,7 +99,7 @@ For more information, see "[Reusing workflows](/actions/learn-github-actions/reu
 
 #### `on.workflow_call.inputs.<input_id>.type`
 
-Required if input is defined for the `on.workflow_call` keyword. The value of this parameter is a string specifying the data type of the input. This must be one of: `boolean`, `number`, or `string`.
+Required if input is defined for the `on.workflow_call` keyword. The value of this parameter is a string specifying the data type of the input. This must be one of: `boolean`, `number`, `string`, `environment` or `choice`.
```

Comment:
@isaranto this is related to workflow_call to pass inputs for reusable workflows
The blog [post](https://github.blog/changelog/2021-11-10-github-actions-input-types-for-manual-workflows/) you linked is related to workflow_dispatch which is for manually triggering inputs.